weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallDetails.tsx

```diff
@@ -258,7 +279,9 @@ const getDisplayInputsAndOutput = (call: CallSchema) => {
   const span = call.rawSpan;
   const inputKeys =
     span.inputs._keys ??
-    Object.keys(span.inputs).filter(k => !k.startsWith('_') || k === '_type');
+    Object.keys(span.inputs ?? {}).filter(
```

Comment:
shouldn't inputs be empty dict even when we don't select it?

Reply:
er, yes, but currently we rely on the request columns entirely. For **all** fields. We would have to add some required column logic in the stream builder.

Reply:
so this is a bug in the server?

Reply:
the return type of `inputs` from the server is always a dictionary

Reply:
or it should be

Reply:
Okay this is pretty dumb but essentially what is required: https://github.com/wandb/core/pull/23508. @tssweeney